js/clinicalTimeline.js

```diff
@@ -163,16 +152,30 @@ var clinicalTimeline = (function(){
       .attr("width", overviewAxisWidth)
       .attr("class", "overview");
 
-    clinicalTimelineOverviewAxis(overviewSVG, getTickValues, minDays, maxDays, overviewAxisWidth, formatTime, daysToTimeObject, getZoomLevel, width,  margin);
-    clinicalTimelineVerticalLine(timelineBool.enableVerticalLine, beginning, ending, timelineBool.advancedView, timelineBool.tooltipOnVerticalLine);
-    initHelperLines(timelineBool.enableHelperLines, zoomFactor, timelineBool.advancedView, divId, formatTime, daysToTimeObject);
-
-    if(timelineBool.advancedView){
-      initAdvancedView(overviewSVG)
-    } else {
-      initSimpleView(overviewSVG);
+    //object to be shared by all plugins
+    clinicalTimelineVar = {
+      divId: divId,
+      beginning: beginning,
+      ending: ending,
+      minDays: minDays,
+      maxDays: maxDays,
+      width: width,
+      margin: margin,
+      zoomFactor: zoomFactor,
+      chart: chart,
```

Comment:
I think all these above should just have `get` methods on timeline. Also some of these are settable such as `divId`, but some other such as `minDays` and `maxDays` are not, which is a little confusing.